pkg/dao/repository_config_test.go

```diff
@@ -1044,7 +1044,7 @@ func (suite *RepositoryConfigSuite) TestValidateParameters() {
 		Timeout: true,
 	}
 
-	mockExtRDao.Mock.On("ValidRepoMD", "http://timemeout.com").Return(0, timeoutErr)
+	mockExtRDao.Mock.On("ValidRepoMD", "http://timemeout.com/").Return(0, timeoutErr)
```

Comment:
I could not find a special use domain name that provides a time out. Hope the domain does not change hands soon.

Reply:
we could just use timeout.example.com. In this instance the actual domain doesn't matter